app/views/layouts/application.haml

```diff
@@ -22,10 +22,10 @@
     %meta{name: "application-name", content: "Glowfic Constellation"}
     %meta{name: "msapplication-config", content: asset_path('favicons/browserconfig.xml')}
     %meta{name: "theme-color", content: "#ffffff"}
-    = favicon_link_tag asset_path('favicons/favicon.ico')
-    = favicon_link_tag asset_path('favicons/favicon-32x32.png'), type: 'image/png', rel: 'icon', sizes: '32x32'
-    = favicon_link_tag asset_path('favicons/favicon-16x16.png'), type: 'image/png', rel: 'icon', sizes: '16x16'
-    = favicon_link_tag asset_path('favicons/apple-touch-icon.png'), type: 'image/png', rel: 'apple-touch-icon', sizes: '180x180'
+    = favicon_link_tag 'favicons/favicon.ico'
```

Comment:
This routes through `/assets/favicons/favicon.ico` for me in dev mode, which does serve the appropriate asset.

The exact error, on this line:

```haml
= favicon_link_tag asset_path('favicons/favicon.ico')
```

is:

> Asset names passed to helpers should not include the "/assets/" prefix. Instead of "/assets/favicons/favicon.ico", use "favicons/favicon.ico"
> app/views/layouts/application.haml:25:in '_app_views_layouts_application_haml__3992509873192304109_46982837423520'
app/controllers/application_controller.rb:78:in 'set_timezone'

I assume this is because `asset_path` autogenerates the prefix, and it doesn't look like it's adding a hash, so I'd expect this change to make it work as it suggests.